### PR TITLE
FADE

### PR DESCRIPTION
**Pull request by rdvelazquez**
FADE is now working (you can try it out at http://newmethods.datamonkey.org/)

### CODE REVIEW

File: app/models/msa.js [hunk 1]
```diff
@@ -240,7 +240,8 @@ Msa.methods.aminoAcidTranslation = function(cb, options) {
 
 Msa.methods.dataReader = function(file, datatype, cb) {
   // Skip the datareader batch file for fastq and non-coding DNA/RNA files.
-  if (file.indexOf("fastq") != -1 || datatype == 1) {
+  console.log("datatype: ", datatype);
```
Comment (stevenweaver) [inline app/models/msa.js]: Using winston here would give us better control over what is printed to logs.

File: app/routes/fade.js [hunk 2]
```diff
@@ -1,205 +1,136 @@
...
-exports.createForm = function(req, res) {
-  res.render("fade/upload_msa.ejs");
-};
-
-exports.uploadFile = function(req, res) {
-  var PROTEIN_GENCODE = -2;
-  var data = req.body;
-  var fn = req.files.files.file;
+var redis = require("redis"),
+  client = redis.createClient({ host: "localhost", port: 6379 });
```
Comment (stevenweaver) [inline app/routes/fade.js]: Maybe I'm a bad example in other parts of the code, but the redis port should be configurable instead of hardcoded.

Comment (stevenweaver) [reply app/routes/fade.js]: As well as the host.

File: src/jsx/forms/fubar.jsx [hunk 3]
```diff
@@ -149,6 +149,7 @@ class FUBARForm extends React.Component {
 
     xhr.onload = function(res) {
       // Replace field with green text, name of file
+      console.log("this.responseText: ", this.responseText);
```
Comment (stevenweaver) [inline src/jsx/forms/fubar.jsx]: stray console log

File: src/jsx/stats.jsx [hunk 4]
```diff
@@ -412,6 +412,7 @@ var UsageDashboard = React.createClass({
         "absrel",
         "busted",
         "relax",
+        "fade",
```
Comment (stevenweaver) [inline src/jsx/stats.jsx]: 👌🏻

File: app/models/msa.js (hunk 1 above)
Comment (rdvelazquez) [reply app/models/msa.js]: This was just a stray console.log from debugging 😞

File: app/routes/fade.js (hunk 2 above)
Comment (rdvelazquez) [reply app/routes/fade.js]: Makes sense. My last commit addressed this. Update http://newmethods.datamonkey.org/ to ensure it works.

Comment (rdvelazquez) [reply app/routes/fade.js]: I updated http://newmethods.datamonkey.org/ and successfully ran a SLAC and FADE job to make sure the change worked 👍

**Pull request merged**